rpc/src/main/scala/proto/model.scala

```diff
@@ -193,7 +193,10 @@ object model {
   final case class ProtoOption(name: String, value: String, quote: Boolean)
 
   final case class ProtoDefinitions(
-      prelude: String = "syntax = \"proto3\";",
+      prelude: String = """|// This file has been automatically generated for use by
```

Comment:
Add link to frees.io future doc section

Reply:
👍 

https://github.com/frees-io/freestyle-rpc/issues/51